WcaOnRails/app/assets/stylesheets/competitions.scss

```diff
@@ -150,3 +182,67 @@ $venue-map-wrapper-height: 400px;
 .big-column {
   width: 100%;
 }
+
+.competition-select.form-inline {
+  background-color: #fff;
+  width: 100%;
+  padding-bottom: 10px;
+
+  input,
+  select {
+    font-size: 12px;
+    font-weight: normal;
+    display: block;
+  }
+
+  button {
+    width: 100px;
+  }
+
+  .btn-group {
+    display: block;
+  }
+}
+
+.same-line {
+  display: inline-block;
+  margin-right: 5px;
+}
+
+#today-line {
+  @extend .bg-primary;
+}
+
+#scroller {
```

Comment:
Can we give this a better name than `scroller`? This is the competition query form holder, right?